### PR TITLE
[release-2.1 < T0739-MG] Add docs for bootstrap option

### PR DESCRIPTION
**Pull request by kostasrim**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.0.2 - TBA 
+* Added `BOOTSTRAP_SERVERS` option to `CREATE STREAM` which you can check [here.](reference-guide/streams/overview.md)
```
Comment (antonio2368) [inline docs/changelog.md:8]: Add PR link which introduced that change. Also rebase.

**Review by g-despot — APPROVED** [review]

File: docs/changelog.md (hunk 1 above)
Comment (kostasrim) [reply docs/changelog.md]: done!

File: docs/changelog.md [hunk 2]
```diff
@@ -17,6 +18,7 @@ sidebar_label: Changelog
   query. [#278](https://github.com/memgraph/memgraph/pull/278)
 * Fix linkage error when `mgp_local_time_get_minute` is used. [#273](https://github.com/memgraph/memgraph/pull/273)
 * Fix crash when temporal types are used with `ORDER BY` clause. [#299](https://github.com/memgraph/memgraph/pull/299)
+* Added `BOOTSTRAP_SERVERS` option to `CREATE STREAM` which you can check [#282](reference-guide/streams/overview.md)
```
Comment (antonio2368) [inline docs/changelog.md:20]:
```suggestion
* Added `BOOTSTRAP_SERVERS` option to `CREATE KAFKA STREAM` which you can check [here](reference-guide/streams/overview.md). [#282](https://github.com/memgraph/memgraph/pull/282)
```

**Comment by kostasrim** [comment]
@antaljanosbenjamin take a look [here](https://memgraph.com/docs/memgraph/database-functionalities/streams/kafka-streams#creating-the-stream)

**Review by antonio2368 — APPROVED** [review]

**Review by antaljanosbenjamin — APPROVED** [review]

**Pull request merged**